src/integer/intReduce.sml

```diff
@@ -319,16 +320,16 @@ end (* local *)
 
 local
   val pth0 = prove
-     (“(&0 * &x = &0) /\
-       (&0 * --(&x) = &0) /\
```

Comment:
This actually fixed a porting bug of `INT_MUL_CONV` on the input of the form `&0 * x`: in HOL-Light the integer subtraction const is `--` instead of the overloaded `-`.